### ISSUE TITLE
Add alternate text

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#105003]
## Background
We had a finding from our 508 audit of Appoint a Representative 1.0.

> “Alert” icon is missing meaningful text equivalent.
> Defect 1 | Critical | Created:
> WCAG 1.1.1 Non-text Content | All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below.
> https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/representative-contact -> 1 of 5 Accredited representative information.
> The “Alert” graphic is not equivalent to the information depicted visually. For example, as a JAWS user, for step “1 of 5 Accredited representative information”, when the user reads the content in the “Alert” section, the JAWS user dose not read the text equivalent of the icon before alerting the user to contact the accredited representative. (New: 02/24/2025).

After consulting Platform's accessibility team, the guidance to address is to add an aria label so the "1" is read as "You are on step 1" for screen readers.

<img width="1466" alt="Image" src="https://github.com/user-attachments/assets/2507e50c-dcc9-4a97-a749-cf14a190e3ca" />

## Tasks
- [ ] Add an aria label "You are on step x" to the active step (in the blue circle) on each page of Appoint a Representative

## Acceptance Criteria
- [ ] The step guidance on Appoint a Representative is accessible to all.

**Comment by humancompanion-usds** [comment]
I asked @jeana-adhoc to test this (@rsmithadhoc is our a11y specialist on the Design System team, but he's on vacation at the moment). Jeana found that:

"It does announce "Step 2 of 5" What it doesn't announce, and what the 508 office is saying is that it should say "You are on step 2 of 5""

We both agree this is splitting hairs and not a blocker. However, it is a simple change to make that we can make. 

@caw310 - Can you either transfer this issue to our repo or duplicate it so that we can fix it? Thanks!

**Comment by oddball-lindsay** [comment]
Calling out that I had the wrong finding details quoted above, the issue description is now fixed to reference the correct finding.
